src/core/system_parameters/Beidou_B1C.h

```diff
@@ -34,16 +34,32 @@
 #define GNSS_SDR_BEIDOU_B1C_H_
 
 
+#include "MATH_CONSTANTS.h"
+#include "gnss_frequencies.h"
+#include <cstdint>
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
 
-const double BEIDOU_B1Cd_CODE_RATE_HZ = 10.23e6;      //!< BEIDOU_B1c data code rate [chips/s]
-const int32_t BEIDOU_B1Cd_CODE_LENGTH_CHIPS = 10230;  //!< BEIDOU_B1c data  code length [chips]
+
+const double BEIDOU_B1C_FREQ_HZ = FREQ1;              //!< BEIDOU B1C carrier frequency [Hz]
+
+
+const double BEIDOU_B1Cd_CODE_RATE_HZ = 10.23e6;      //!< BEIDOU_B1C data code rate [chips/s]
```

Comment:
Please check the datasheet because the code rate for B1C is 1.023e6 instead of 10.23e6.

Reply:
You are right! Updated to correct value.